README.md

```diff
@@ -48,8 +48,7 @@ You can manually drag the fonts from the `fonts/otf` or `fonts/variable` directo
 There is also a script that automates the deletion of all Monaspace fonts from `~/Library/Fonts` and then copies over the latest versions. Invoke it from the root of the repo like:
 
 ```bash
-$ cd util
-$ bash ./install_macos.sh
+$ bash ./util/install_macos.sh
```

Comment:
```suggestion
$ bash util/install_macos.sh
```

Reply:
Can you explain your suggestion? Being explicit about using the current directory is clearer and consistent with the existing style.

Reply:
> Being explicit about using the current directory is clearer

I personally find NOT specifying the current directory clearer ¯\\\_(ツ)_/¯ there's a tiny bit less stuff to parse by our brains

> consistent with the existing style i.e. with this change it would not match the Linux installation instructions

didn't notice 😄 in that case I'd personally also remove it from Linux installation instructions; I simply find it unnecessary as `bash install_linux.sh` already tells me the file is in the cwd :)


I'm fine with it being `bash ./util/install_macos.sh` though, it's really just a nit